zend_abstract_interface/config/php7-8/config_runtime.c

```diff
@@ -20,12 +20,47 @@ void zai_config_replace_runtime_config(zai_config_id id, zval *value) {
     ZVAL_COPY(rt_value, value);
 }
 
+static inline zval *zai_config_runtime_config_value(zai_config_memoized_entry *memoized, zval *updated) {
+    ZAI_ENV_BUFFER_INIT(buf, ZAI_ENV_MAX_BUFSIZ);
+
+    for (int16_t name_index = 0; name_index < memoized->names_count; name_index++) {
```

Comment:
We must skip this loop for system inis. System inis are recognized via `memoized->ini_change == zai_config_system_ini_change`.